src/Schema/Directives/RelationDirective.php

```diff
@@ -32,9 +33,15 @@ abstract class RelationDirective extends BaseDirective implements FieldResolver
      */
     protected $lighthouseConfig;
 
-    public function __construct(ConfigRepository $configRepository)
+    /**
+     * @var DatabaseManager
```

Comment:
```suggestion
     * @var \Illuminate\Database\DatabaseManager
```

Also can we use `Illuminate\Database\ConnectionResolverInterface` instead?

Reply:
yep